cypress/integration/mixins.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('Should unbind event listeners that bound by the Mixins after the map is destroyed', () => {
+  it('KeyboardMixin', () => {
+    cy.window().then((window) => {
+      const { map, document } = window;
```

Comment:
This should be enough:
```suggestion
    cy.window().then(({ map, document }) => {
```

Reply:
Never mind, i didn't see that you need the window object later